sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -97,7 +97,7 @@ internal CryptographyClient(JsonWebKey keyMaterial, TokenCredential credential,
         /// <summary>
         /// The <see cref="Key.Id"/> of the key used to perform cryptographic operations for the client.
         /// </summary>
-        public Uri KeyId => _keyId;
+        public string KeyId => _keyId.ToString();
```

Comment:
> public string KeyId => _keyId.ToString(); [](start = 7, length = 42)

I would rather leave the public property a Uri and explicitly implement the interface:

string IKeyEncryptionKey.KeyId { get { return _keyId.ToString(); } }

Reply:
How does this related to https://github.com/Azure/azure-sdk-for-net/pull/7775#discussion_r329691722? Should indeed keep it a string, or was this only true for the JWK?